covid19-etl/etl/idph_facility.py

```diff
@@ -0,0 +1,183 @@
+            base_url=self.base_url,
+            program_name=self.program_name,
+            project_code=self.project_code,
+            access_token=access_token,
+        )
+
+        self.country = "US"
+        self.state = "IL"
+
+        self.summary_locations = []
+        self.summary_clinicals = []
+
+    def files_to_submissions(self):
+        """
+        Reads JSON file and convert the data to Sheepdog records
+        """
+
+        latest_submitted_date = self.metadata_helper.get_latest_submitted_date_idph()
+        today = datetime.date.today()
+        if latest_submitted_date == today:
+            print("Nothing to submit: today and latest submitted date are the same.")
+            return
+        today_str = today.strftime("%Y%m%d")
+
+        print(f"Getting data for date: {today_str}")
+        url = "https://dph.illinois.gov/sitefiles/COVIDLTC.json"
+        self.parse_file(latest_submitted_date, url)
+
+    def parse_file(self, latest_submitted_date, url):
+        """
+        Converts a JSON files to data we can submit via Sheepdog. Stores the
+        records to submit in `self.summary_locations` and `self.summary_clinicals`.
+        :param latest_submitted_date: the date of latest available "summary_clinical" for project
+        :param url: URL of JSON for parsing
+        :return: fast-return if no new data is available
+        """
+        print("Getting data from {}".format(url))
+        with closing(requests.get(url, stream=True)) as r:
+            data = r.json()
+            date = self.get_date(data)
+
+            if latest_submitted_date and date == latest_submitted_date.strftime(
+                "%Y-%m-%d"
+            ):
+                print(
+                    "Nothing to submit: latest submitted date and date from data are the same."
+                )
+                return
+
+            summary_location_submitter_id = format_summary_location_submitter_id(
+                self.country, self.state
+            )
+            summary_location = {
+                "country_region": self.country,
+                "submitter_id": summary_location_submitter_id,
+                "projects": [{"code": self.project_code}],
+                "province_state": self.state,
+            }
+
+            summary_clinical_submitter_id = format_summary_clinical_submitter_id(
+                summary_location_submitter_id, date
+            )
+            summary_clinical = {
+                "confirmed": data["LTC_Reported_Cases"]["confirmed_cases"],
+                "deaths": data["LTC_Reported_Cases"]["deaths"],
+                "submitter_id": summary_clinical_submitter_id,
+                "lastUpdateEt": date,
+                "date": date,
+                "summary_locations": [{"submitter_id": summary_location_submitter_id}],
+            }
+            self.summary_locations.append(summary_location)
+            self.summary_clinicals.append(summary_clinical)
```

Comment:
are these 2 totals for all the facilities?

Reply:
The below loop are for facilities, and this is for state-level `summary_location` & `summary_clinical` for this data.